airflow/www_rbac/templates/airflow/couture_config.html

```diff
@@ -398,186 +320,118 @@ <h3>Configurations</h3>
-    for(var i=0; i<selected_py.length; i++){
-        y=selected_py[i];
-        file_name = y.split('\\').pop().split('/').pop();
-        selected[i]=file_name;
-        }
-    var myDiv = document.getElementById("Py_couturecheckboxes");
-    if (Py_count_of_clicks == 1)
-    {
-        var checkbox = "";
-        for(var t=0; t<files.length; t++) {
-            file_name=files[t];
-            var py_checkBox = document.createElement("input");
-            var label = document.createElement("label");
-            var in_selected = false;
-            for (var i=0;i<selected.length;i++)
-            {
-                if(file_name == selected[i]) {
-                    in_selected= true;
-                }
-            }
-            if (in_selected==true) {
-                py_checkBox.checked = "true";
-            }
-            py_checkBox.type = "checkbox";
-            py_checkBox.value = file_name;
-            py_checkBox.name = "py_check";
-            py_checkBox.id = "py_checkBox";
-            py_checkBox.onclick = "javascript:showValue()";
-            label.for = file_name;
-            myDiv.appendChild(label);
-            label.appendChild(py_checkBox);
-            py_checkBox.appendChild(document.createTextNode(file_name));
-            label.appendChild(document.createTextNode(file_name));
-        }
-  }
 
-}
-var kt_expanded = false;
-kt_count_of_clicks=1
 function kt_showcouturecheckboxes(f_3) {
   kt_count_of_clicks++;
   var kt_couturecheckboxes = document.getElementById("kt_couturecheckboxes");
-  if (!expanded) {
+  if (!kt_expanded) {
     kt_couturecheckboxes.style.display = "block";
-    expanded = true;
+    kt_expanded = true;
   } else {
     kt_couturecheckboxes.style.display = "none";
-    expanded = false;
+    kt_expanded = false;
   }
   if(f_3) {
     kt_couturecheckboxes.style.display = "none";
-    expanded = false;
+    kt_expanded = false;
   }
 }
 
-function kt_getAllNames(files,selected_kt) {
-    var selected_kt = selected_kt.split(',');
-    selected=[]
-    for(var i=0; i<selected_kt.length; i++){
-        y=selected_kt[i];
-        file_name = y.split('\\').pop().split('/').pop();
-        selected[i]=file_name;
-
-        }
-    var myDiv = document.getElementById("kt_couturecheckboxes");
-    if (kt_count_of_clicks == 1)
-    {
-        var checkbox = "";
-        for(var t=0; t<files.length; t++) {
-            file_name=files[t];
-            var kt_checkBox = document.createElement("input");
-            var label = document.createElement("label");
-            var in_selected = false;
-            for (var i=0;i<selected.length;i++)
-            {
-                if(file_name == selected[i]) {
-                    in_selected= true;
-                }
-            }
-            if (in_selected==true) {
-                kt_checkBox.checked = "true";
+//function that gets the names of files and puts it onto the required labels
+function getAllNames_test(files,selected_files,num) {
```

Comment:
change the function name. Also, either use camelCase or snake_case. Don't use both. For function names, use camelCase.